### PR TITLE
use print function from the future

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

File: src/odemis/model/_dataflow.py [hunk 1]
```diff
@@ -586,18 +585,19 @@ def run(self):
                         self.w_notifier(darray)
                     except WeakRefLostError:
                         return  # It's a sign there is nothing left to do
-        except:
+        except Exception:
```
Comment (philipwinklerdelmic) [inline src/odemis/model/_dataflow.py]: Why did you change it?

Comment (pieleric) [reply src/odemis/model/_dataflow.py]: Indeed, that's a little extra, not related to `print()`. As I was already touching that file for non-behavioural changes, I took the opportunity for that clean-up too.

Apparently using `except` without a specific type of Exception is not good practice, as it may catch a few very special exceptions which should not really ever be caught. Using `except Exception` is considered a bit better... although the linter still find it too general. In that case, the whole point is to be generic, so the linter is just not smart enough.

Comment (philipwinklerdelmic) [reply src/odemis/model/_dataflow.py]: I had no idea... Just using except also catches system exit exceptions like keyboard interrupt. I would have guessed that it's good practice to catch everything (so using except and not except Exception). Also, I've never seen people use it before, but that could just mean that I haven't read much python code...
